watertap/costing/watertap_costing_package.py

```diff
@@ -1140,10 +1232,10 @@ def cost_gac(blk):
             expr=blk.gac_makeup_cost
             == pyo.units.convert(
                 (
-                    blk.costing_package.gac_makeup_unit_cost
+                    blk.costing_package.gac.makeup_unit_cost
                     * (
-                        (1 - blk.costing_package.gac_regen_frac)
-                        * blk.unit_model.gac_mass_replace_rate
+                        (1 - blk.costing_package.gac.regen_frac)
+                        * blk.unit_model.gac.mass_replace_rate

```

Comment:
```suggestion
                        * blk.unit_model.gac_mass_replace_rate
```

A unit_model variable with the prefix `gac_` got changed when find and replacing `costing.gac_` variables